Scripts/RemediationScripts/Remediate-EnableEncryptionInTransitForStorageAccounts.ps1

```diff
@@ -0,0 +1,691 @@
+        $userInput = Read-Host -Prompt "(Y|N)"
+    
+        if($userInput -ne "Y")
+        {
+            Write-Host "secure transfer will not be disabled for any of the storage account. Exiting..." -ForegroundColor $([Constants]::MessageType.Info)
+            break
+        }
+    }
+    else
+    {
+        Write-Host "'Force' flag is provided. secure transfer will be disabled on the storage account without any further prompts." -ForegroundColor $([Constants]::MessageType.Warning)
+    }
+    Write-Host $([Constants]::SingleDashLine)
+
+    Write-Host "[step 4 of 4]: Disable secure transfer on the Storage Account in the Subscription."
+    
+    try
+    {
+        if(($remediatedResourceLog | Measure-Object).Count -gt 0)
+        {   $rollbackSuccess = @()
+            $rollbackFailure = @()
+                    
+            Write-Host "Disabling secure transfer on [$(($remediatedResourceLog| Measure-Object).Count)] Storage Account(s) of subscription [$($SubscriptionId)]..." 
+           
+            $remediatedResourceLog | ForEach-Object {
+                try
+                {
+                    $output = Set-AzStorageAccount -ResourceGroupName $_.ResourceGroupName -Name $_.StorageAccountName -EnableHttpsTrafficOnly $false -ErrorAction SilentlyContinue
+                    if($output -ne $null)
+                    {
+                        $rollbackSuccess += $_ |Select-Object -Property "StorageAccountName" , "ResourceGroupName" , "ResourceId"
+                    }
+                    else
+                    {  
+                       Write-Host "Rollback is not successful on [$($_.StorageAccountName)] : [$($_.ResourceGroupName)]" -ForegroundColor $([Constants]::MessageType.Warning)    
+                       $rollbackFailure += $_ |Select-Object -Property "StorageAccountName" , "ResourceGroupName" , "ResourceId"                              
+                    }
+                }             
+                catch
+                {
+                    Write-Host "Error occurred while rollback on [$($_.StorageAccountName)] : [$($_.ResourceGroupName)]" -ForegroundColor $([Constants]::MessageType.Error)
+                    Write-Host  "Error: $($_)" -ForegroundColor $([Constants]::MessageType.Error)
+                    $rollbackFailure += $_ |Select-Object -Property "StorageAccountName" , "ResourceGroupName" , "ResourceId"
+                }
+            }
+
+            Write-Host $([Constants]::DoubleDashLine)
+
+            if(($rollbackSuccess|Measure-Object).Count -ne 0)
+            {   Write-Host "Rollback is successful on following  Storage Account(s):" -ForegroundColor ([Constants]::MessageType.Update)
+                $rollbackSuccess | Select-Object -Property "ResourcegroupName", "storageAccountName" ,"ResourceId" | ft   
+                write-host $([Constants]::SingleDashLine)
```

Comment:
`write-host` to be replaced with `Write-Host`.
Please delete the stray newline.